editor/src/clj/editor/fxui.clj

```diff
@@ -502,32 +501,51 @@
       (assoc :fx/type fx.tooltip/lifecycle)
       (util/provide-defaults :wrap-text true :max-width 600)))
 
-(def ^:private javafx-property-lifecycle
-  (reify fx.lifecycle/Lifecycle
-    (create [_ desc _]
-      (SimpleObjectProperty. desc))
-    (advance [_ component desc _]
-      (doto ^SimpleObjectProperty component (.set desc)))
-    (delete [_ _ _])))
-
 (defonce ^:private custom-tooltip-node-showing-key (Object.))
 
 (def prop-custom-tooltip-node-showing
   (fx/make-prop
-    (fx.mutator/adder-remover
-      (fn [^Node node value]
-        (.put (.getProperties node) custom-tooltip-node-showing-key value))
-      (fn [^Node node _]
-        (.remove (.getProperties node) custom-tooltip-node-showing-key)))
-    javafx-property-lifecycle))
+    (fx.mutator/setter
+      (fn set-custom-tooltip-node-showing [^Node node value]
+        (.put (.getProperties node) custom-tooltip-node-showing-key value)))
+    fx.lifecycle/scalar))
+
+(defn- observable-map+key->observable-val
```

Comment:
Naming: We typically use `something+other` to refer to tuples. Maybe it would be clearer as something like `make-observable-value-for-key`?